### DISCUSSION TITLE
Live microphone transcription in python (Pycharm)

### DISCUSSION THREAD

**Discussion by ZelenskyGigaChad** [deepgram/community#337] [General help] [answered]
Hello, i'm making a project for my school, and I can't seem to find how to use the deepgram stt model with my microphone.

**Comment by jjmaldonis** [comment]
Hey @ZelenskyGigaChad, this sounds like a cool project!

If you want to record through the browser, you can check out our Node tutorial here: https://deepgram.com/learn/live-transcription-mic-browser

If you're using Python and do not want to use the browser, check out this tutorial: https://deepgram.com/learn/how-to-transcribe-only-what-you-need-with-python-listening-before-connected

Let us know how it goes and if you run into any obstacles :)

**Reply by ZelenskyGigaChad** [reply]
Hello! Thank you very much for the help! I will be using deepgram stt through pycharm so i'm using the code from the second tutorial : https://deepgram.com/learn/how-to-transcribe-only-what-you-need-with-python-listening-before-connectedHow
However i'm getting an error in these lines (52);
while True:
data = await audio_queue.get().
await ws.send(data)
Is the comma at the end of await audio_queue.get() a missclick? I tried to remove it, but it still wasn't working. Thank you very much in advance!

**Reply by jjmaldonis** [reply, marked as answer]
Yes, the period at the end of the `await audio_queue.get().` is a typo. I got your email, let's sync there.
